### PR TITLE
Remove create database options completely

### PR DESCRIPTION
**Pull request by XisiHuang**

### CODE REVIEW

**Comment by tschottdorf** [comment]
LGTM. @petermattis do we want to keep the original rules around (so that reintroducing parts of them later doesn't get awkward) or is it ok to just toss things out?

File: sql/parser/sql.y [hunk 1]
```diff
@@ -1536,22 +1536,15 @@ transaction_mode_list_or_empty:
...
-    // TODO(pmattis): Handle options.
     $$ = &CreateDatabase{IfNotExists: true, Name: Name($6)}
   }
 
-createdb_opt_with:
-  WITH {}
-| WITH_LA {}
-| /* EMPTY */ {}
-
 createdb_opt_list:
```
Comment (petermattis) [inline sql/parser/sql.y]: I think you can remove `createdb_opt_list`, `createdb_opt_items` and `createdb_opt_item` as well.

**Comment by petermattis** [comment]
LGTM

@tschottdorf I think it is best to remove the rules entirely for these non-core rules. Rules that affect `{SELECT,INSERT,UPDATE,DELETE}` we should be more careful with

File: sql/parser/sql.y (hunk 1 above)
Comment (XisiHuang) [inline sql/parser/sql.y]: done.

**Comment by petermattis** [comment]
LGTM

**Comment by tamird** [comment]
@XisiHuang merge whenever you're ready.

**Comment by XisiHuang** [comment]
please merge, thanks. @tamird

**Comment by tschottdorf** [comment]
@XisiHuang you can merge it yourself.

**Pull request merged**